src/commands/sourcemaps/inject.rs

```diff
@@ -41,7 +41,8 @@ pub fn execute(matches: &ArgMatches) -> Result<()> {
         let entry = match entry {
             Ok(entry) => entry,
             Err(e) => {
-                error!("{e}");
+                debug!("skipping file {path}");
```

Comment:
This is path used to recursively look for files, but not the entry that errored 🤔

Reply:
Derp

Reply:
The error returned by `walkdir` contains the path. What do you think of output like this?
```
IO error for operation on inject_test/forbidden: Permission denied (os error 13), skipping file
```

Reply:
Oh, I see they have accessors for it as well - https://docs.rs/walkdir/latest/walkdir/struct.Error.html#example
But tbh the error they produce is nice enough. I'd just reverse the order to be "action: cause" instead.

```
Skipping file: {e}
```

Reply:
Or just `Skipping file: {e}`, either is fine